R/constants.R

```diff
@@ -149,3 +149,12 @@ emissions.TST_TO_TG <- 0.000907 # Conversion from thousand short tons to Tg
 emissions.NH3_HISTORICAL_YEARS <- 1990:2002
 emissions.NH3_EXTRA_YEARS <- 1971:1989
 emissions.EDGAR_YEARS <- 1971:2008
+
+# ======================================================================
+# Gas technologies
+
+energy.GAS_TECH <- c( "gas (steam/CT)", "gas (CC)" )
+
```

Comment:
I think the gas technology names must come from the input file (energy/A23.globaltech_eff) rather than being hardcoded here. Otherwise, if the name changed in the input file so that it is inconsistent with above, there would be a problem.